### PR TITLE
create muhalvin.json

### PR DESCRIPTION
**Pull request by muhalvin**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/muhalvin.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "For my portofolio and personal project",
+    "repo": "https://github.com/muhalvin/muhalvin.github.io",
+    "owner": {
+        "username": "muhalvin",
+        "email": "work.muhalvin@gmail.com  ",
+    },
+    "record": {
+        "CNAME": "muhalvin.github.io"
```
Comment (CuteDog5695) [inline domains/muhalvin.json:9]: Please add more stuff to your website

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/muhalvin.json (hunk 1 above)
Comment (muhalvin) [reply domains/muhalvin.json:9]: done sir, i've updated my website just now.

Comment (muhalvin) [reply domains/muhalvin.json:9]: image


muhalvin.github.io

File: domains/muhalvin.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "For my portofolio and personal project",
+    "repo": "https://github.com/muhalvin/muhalvin.github.io",
+    "owner": {
+        "username": "muhalvin",
+        "email": "work.muhalvin@gmail.com  ",
```
Comment (CuteDog5695) [inline domains/muhalvin.json:6]:
```suggestion
        "email": "work.muhalvin@gmail.com"
```

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Comment by muhalvin** [comment]
Thank you for your time, appreciate it very much.

**Pull request merged**